core/src/main/java/com/rim/IceField/PlayerBase.java

```diff
@@ -2,52 +2,54 @@
 public abstract class PlayerBase {
     protected String tag;
     protected int ID;
-    protected boolean isWearingDSuit;
+    protected boolean isWearingDSuit = false;
     protected int heatLevel;
-    protected boolean isDead;
+    protected boolean isDead = false;
     protected int numOfMoves;
     public Inventory Inv;
 
+    /** The constructor for the playerbase just
+     *
+     */
     public PlayerBase()
     {
        this.tag = "PlayerBase";
+       Inv = new Inventory();
     }
 
     public void move()
     {
 
     }
 
-    public void SavePlayer()
+    public void SavePlayer(PlayerBase player)
     {
 		System.out.println("Player has been saved!");
     }
 
     public void useSkill()
     {
-		if (this.tag.equals("Eskimo"))
-		{
-			System.out.println("An Igloo has been created!");
-		} else if (this.tag.equals("PolarExplorer"))
-		{
-			//Not sure if this is correct
-			System.out.println("Neighboring icebergs have been checked!");
-		}
+        System.out.println("Empty Skill.");
     }
 
     public void useItem()
     {
-
+        //Not sure how I would check a specific item in the inventory here.
     }
 
     public void die()
     {
 		System.out.println("You have died. RIP ):");
     }
 
+    /** In the fall() method we decrement the current heatLevel by 1, and that will be updated every second,
+     * Meaning eskimos get 5 seconds to live if they fall, and Polar expolorers get 4 seconds.
+     */
+
     public void fall()
     {
 		System.out.println("Ouch! You've fallen into some water");
+		heatLevel -= 1;
```

Comment:
we should think here how to continuously decrease the heat levels..Maybe start a timer in another thread? Not sure although where we need to do that.

Reply:
I think we can just put the call the fall() in the update which would keep executing it. But alternatively we can start a timer. I'll think about it though